integration-tests/features/server_api.feature

```diff
@@ -31,9 +31,327 @@ Feature: Server API
+         |stack_analyses|2-0-0|
+         |stack_analyses|2-0-1|
+         |stack_analyses|2-0-2|
+         |stack_analyses|2-0-3|
+         |stack_analyses|2-1-0|
+         |stack_analyses|2-1-1|
+         |stack_analyses|2-1-2|
+         |stack_analyses|2-1-3|
+         |stack_analyses|2-1-4|
+         |stack_analyses|2-2-0|
+         |version_range_resolver|1-0-0|
+
+  @production
+  Scenario Outline: Check the /api/v1/schemas/component_analyses response
+    Given System is running
+    When I access /api/v1/schemas/component_analyses
+    Then I should get 200 status code
+    Then I should find the schema <schema> version <version> in the list of supported schemas
+
+     Examples: schemas
+         |schema|version|
+         |binary_data|1-0-0|
+         |blackduck|1-0-0|
+         |code_metrics|1-0-0|
+         |crypto_algorithms|1-0-0|
+         |dependency_snapshot|1-0-0|
+         |digests|1-0-0|
+         |github_details|1-0-0|
+         |github_details|1-0-1|
+         |github_details|1-0-2|
+         |github_details|1-0-3|
+         |github_details|1-0-4|
+         |github_details|2-0-0|
+         |github_details|2-0-1|
+         |keywords_tagging|1-0-0|
+         |languages|1-0-0|
+         |metadata|1-0-0|
+         |metadata|1-0-1|
+         |metadata|1-1-0|
+         |metadata|2-0-0|
+         |metadata|2-1-0|
+         |metadata|2-1-1|
+         |metadata|3-0-0|
+         |metadata|3-0-1|
+         |metadata|3-1-0|
+         |metadata|3-1-1|
+         |metadata|3-2-0|
+         |package_keywords_tagging|1-0-0|
+         |security_issues|1-0-0|
+         |security_issues|2-0-0|
+         |security_issues|3-0-0|
+         |security_issues|3-0-1|
+         |source_licenses|1-0-0|
+         |source_licenses|2-0-0|
+         |source_licenses|3-0-0|
+
+  @production
+  Scenario Outline: Check the /api/v1/schemas/api/<name> response
+    Given System is running
+    When I access /api/v1/schemas/api/<schema>
+    Then I should get 200 status code
+    Then I should find the schema version <version> in the list of schema versions
+
+     Examples: schemas
+         |schema|version|
+         |analyses_graphdb|1-0-0|
+         |analyses_graphdb|1-1-0|
+         |analyses_graphdb|1-2-0|
+         |component_analyses|1-0-0|
+         |component_analyses|1-0-1|
+         |component_analyses|1-1-0|
+         |component_analyses|1-1-2|
+         |component_analyses|1-1-3|
+         |stack_analyses|1-0-0|
+         |stack_analyses|1-1-0|
+         |stack_analyses|1-2-0|
+         |stack_analyses|2-0-0|
+         |stack_analyses|2-0-1|
+         |stack_analyses|2-0-2|
+         |stack_analyses|2-0-3|
+         |stack_analyses|2-1-0|
+         |stack_analyses|2-1-1|
+         |stack_analyses|2-1-2|
+         |stack_analyses|2-1-3|
+         |stack_analyses|2-1-4|
+         |stack_analyses|2-2-0|
+         |version_range_resolver|1-0-0|
+
+  @production
+  Scenario Outline: Check the /api/v1/schemas/component_analyses response
```

Comment:
How is this different than [this](https://github.com/fabric8-analytics/fabric8-analytics-common/pull/478/files#diff-72c5f2ddc261b45b0233852128db1761R174)

Reply:
that's a mistake made by me (heh double copy&paste ;). Sorry, I'm fixing it